### PR TITLE
Expand use of customText and refactor constants

### PR DESCRIPTION
**Pull request by maharielrosario**
## Description

CustomText Feature PR Notes:

- created an `appAction` key inside of `customText` to describe actions taken for the type of application
- use  `formConfig.customText.appType` for words like "form" and "application".
- refactor to one `constants` file instead of multiple `constants`

## Testing done
Updated tests; all tests passing

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by rianfowler** [comment]
@mr0sari0 these changes need approval from the content team.

@peggygannon This changes some of the default copy on forms to use the word "application" instead of "form." For example:

> Deleting your form.

becomes 

> Deleting your application.

In other places it won't change the default copy. For example we already use 

> Checking your application status. 

For this specific app they're adding new copy to better match what the veteran is doing. For example

> Deleting your application.
> Application has been saved.

will be changed to

> Deleting your order.
> Order has been saved.

**Comment by mr0sari0** [comment]
@peggygannon I noticed that "form" and "application" were being used interchangeably within the same paragraphs. This PR would set the default to be "application".

Some options we can do are either choose one word to go with or if we still want to have both "form" and "application" used in the same paragraph, I can create a special key for those words. I would just need a good description for what "form" is and what "application" is.

File: src/platform/forms/save-in-progress/ApplicationStatus.jsx [hunk 1]
```diff
@@ -150,7 +155,7 @@ export class ApplicationStatus extends React.Component {
             </p>
             {multipleForms && (
               <p className="no-bottom-margin">
-                You have more than one in-progress {formType} form.{' '}
+                You have more than one in-progress {formType} {appType}.{' '}
```
Comment (mr0sari0) [inline src/platform/forms/save-in-progress/ApplicationStatus.jsx:158]: @rianfowler This is the first I'm seeing of this `formType`. I wasn't able to debug it in the app; do you know what this is supposed to be?

**Review by Clark-W-Griswold — APPROVED** [review]

**Review by bradl3yC — APPROVED** [review]

**Comment by peggygannon** [comment]
hi, @mr0sari0 thanks for checking in. We should be consistent and have the default be "application"

**Comment by mr0sari0** [comment]
> hi, @rianfowler and @mr0sari0 thanks for checking in. Yes, we should be consistent and have the default be "application"
> 
> Can we update "Order has been saved." TO "Your order has been saved"?

@peggygannon It is current set to say "Your order has been saved".

File: src/platform/globalContent.js [hunk 2]
```diff
@@ -8,3 +8,4 @@ export const UNAUTH_SIGN_IN_DEFAULT_MESSAGE =
   'Sign in to start your application';
 export const APP_TYPE_DEFAULT = 'application';
 export const APP_ACTION_DEFAULT = 'applying';
+export const REVIEW_APP_DEFAULT_MESSAGE = 'Review Application';
```
Comment (Clark-W-Griswold) [inline src/platform/globalContent.js:11]: Could we please revert the move and rename of this file?

File: src/platform/forms-system/src/js/constants.js [hunk 3]
```diff
@@ -1,2 +0,0 @@
-export const REVIEW_APP_DEFAULT_MESSAGE = 'Review Application';
-export const APP_TYPE_DEFAULT = 'application';
```
Comment (Clark-W-Griswold) [inline src/platform/forms-system/src/js/constants.js]: Please see other comment regarding constant file that was moved and renamed.

**Review by Clark-W-Griswold — CHANGES_REQUESTED** [review]

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/platform/globalContent.js (hunk 2 above)
Comment (mr0sari0) [reply src/platform/globalContent.js]: I undid the move and renaming.

File: src/platform/forms-system/src/js/constants.js (hunk 3 above)
Comment (mr0sari0) [reply src/platform/forms-system/src/js/constants.js]: Done.

**Review by Clark-W-Griswold — CHANGES_REQUESTED** [review]
It looks like we are moving the `platform/forms/save-in-progress/constants.js` to the `platform/monitoring` folder. Is there a reason we are doing that?

**Comment by mr0sari0** [comment]
> It looks like we are moving the `platform/forms/save-in-progress/constants.js` to the `platform/monitoring` folder. Is there a reason we are doing that?

The save-in-progress constants are being referenced in the `forms` directory. See [here](https://github.com/department-of-veterans-affairs/vets-website/blob/mr0sari0-mdot-content-updates/src/platform/forms/constants.js).

**Comment by Clark-W-Griswold** [comment]
@mr0sari0 
Could we please revert the move of the constants file?

**Comment by mr0sari0** [comment]
> @mr0sari0
> Could we please revert the move of the constants file?

I added back the constants file in `save-in-progress`.

We'll still need a constants file for the messaging in `monitoring`.

**Comment by Clark-W-Griswold** [comment]
@mr0sari0 
It looks like we have some duplicate constants? Is this intended?

File: src/platform/forms-system/src/js/constants.js [hunk 4]
```diff
@@ -1 +1,11 @@
+export const START_NEW_APP_DEFAULT_MESSAGE = 'Start a new application';
```
Comment (Clark-W-Griswold) [inline src/platform/forms-system/src/js/constants.js:1]: This is in two places. Is this intended?

File: src/platform/monitoring/constants.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+export const START_NEW_APP_DEFAULT_MESSAGE = 'Start a new application';
```
Comment (Clark-W-Griswold) [inline src/platform/monitoring/constants.js:1]: This is in two places. Is this intended?

**Review by Clark-W-Griswold — APPROVED** [review]

**Pull request merged**